### ISSUE TITLE
Update Representation Requests segmented button/tablist to use a VADS component

### ISSUE THREAD

**Issue by darbyemadewell** [department-of-veterans-affairs/va.gov-team#137283]
## Background/Context
The segmented button/tablist component is a sort of hybrid component with potentially unexpected keyboard navigation.

The previous recommendation from Platform was to use the `va-segmented-button` component, which didn't exist at the time, so we created a custom component. However, our custom component uses accessiblity attributes for the tabs component and keyboard behavior that aligns more with a segmented button. We should update our implementation to use one of the two components now in VADS: `va-segmented-button` or `va-tabs`. Additional research into this component is needed to identify a which component should be used here.

## Expected vs. Actual
**Expected**: The segmented button/tablist component should align functionality, keyboard behavior, and accessibility attributes to one pattern. It should ideally also use a VADS component now that one exists.

**Actual**: The segmented button/tablist component has functionality and keyboard behavior more aligned to a segmented button, but uses accessibility attributes specific to the tabs pattern. The component is custom and does not use a VADS component.

## Examples

<img width="1624" height="998" alt="Image" src="https://github.com/user-attachments/assets/6f22aee3-da95-4c0c-bb0b-aa430600fc96" />
